scripts/fetch-fonts.mjs

```
@@ -0,0 +1,193 @@
+        `  font-style: ${g.style};`,
+        `  font-weight: ${lo === hi ? lo : `${lo} ${hi}`};`,
+        '  font-display: swap;',
+        `  src: url('./fonts/${local}') format('woff2');`,
+        `  unicode-range: ${g.range};`,
+        '}',
+        ''
+      );
+      totalFaces++;
+    }
+
+    console.log(
```

ESLint

[error] 'console' is not defined.

(no-undef)